spk/sickchill/src/wizard/upgrade_uifile

```diff
@@ -0,0 +1,9 @@
+[{
+    "step_title": "Attention! DSM Permissions",
+    "items": [{
+        "desc": "Permissions for all download-related packages are managed with the group <b>'sc-download'</b> in DSM.<br>Please read <a target=\"_blank\" href=\"https://github.com/SynoCommunity/spksrc/wiki/Permission-Management\">Permission Management</a> for details."
+    },
+    {
+        "desc": "<strong style='color:red'>NOTE:</strong> The package upgrade will try to set the correct permissions on your folders. If you get permission errors within SickChill, manually set Read/Write permissions for the 'sc-download' group on the reported folders using File Station."
+    }]
```

Comment:
I would discard this warning as it concerns only DSM 5 to DSM 6 package migration process